images/linux/scripts/installers/r.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+################################################################################
+##  File:  r.sh
+##  Desc:  Installs R
+################################################################################
```

Comment:
Could you please remove this header because it looks like it doesn't contain any important information?